compiler-plugin/src/main/kotlin/com/google/devtools/ksp/KotlinSymbolProcessingExtension.kt

```diff
@@ -76,6 +76,13 @@ class KotlinSymbolProcessingExtension(
 
                 ServiceLoaderLite.loadImplementations(SymbolProcessorProvider::class.java, classLoader)
             }
+            if (providers.isEmpty()) {
+                logger.error("No providers found in processor classpath.")
+            } else {
+                providers.forEach {
+                    logger.info("loaded provider: ${it.javaClass.name}")
+                }
```

Comment:
Nit: not sure we prefer repeated prefix "loaded provider" for multiple providers? How about just:
```kt
logger.info(
    "loaded provider(s): ${providers.joinToString(separator = ", ", prefix = "[", postfix = "]") { it.javaClass.name }}"
)
```
(or without brackets)

Reply:
that was actually my original implementation (without brackets), I am open to all kinds of suggestions on this matter.